README.Rmd

```diff
@@ -174,7 +177,7 @@ browseVignettes("cleanepi")
 
 ## Next steps
```

Comment:
Also, in the design_principle vignette, some bullet points are not rendered correctly, see for example line 82. Instead of two dashes "--" use only one dash "-". This should be also done in other subsequent lines.